src/Pdk/Product/Repository/WcPdkProductRepository.php

```diff
@@ -98,8 +98,12 @@ public function update(PdkProduct $product): void
     {
         $wcProduct = $this->getWcProduct($product->externalIdentifier);
 
-        $wcProduct->update_meta_data(Pdk::get('metaKeyProductSettings'), $product->settings->toStorableArray());
-        $wcProduct->save();
+        update_metadata(
```

Comment:
hier stond voorheen `update_post_meta`, waarom nu deze functie?

Reply:
`update_post_meta` is een wrapper. Maar we gebruiken nu (weer) de `->save_meta_data` method.